cmd/create/config.go

```diff
@@ -20,9 +20,9 @@ func NewCmdCreateCfg() *cobra.Command {
 		},
 	}
 
-	clusterCfgCmd.Flags().StringVarP(&addons, "add", "", "", "Add plugins")
-	clusterCfgCmd.Flags().StringVarP(&name, "name", "", "", "Name of config file")
-	clusterCfgCmd.Flags().StringVarP(&clusterCfgPath, "config", "f", "", "cluster info config")
+	clusterCfgCmd.Flags().StringVarP(&addons, "add", "", "", "add plugins")
+	clusterCfgCmd.Flags().StringVarP(&name, "name", "", "", "cluster name")
+	clusterCfgCmd.Flags().StringVarP(&clusterCfgPath, "config", "f", "", "configuration file name")
```

Comment:
what does the option look like? --config or -f?

Reply:
It should be modified here. The value of this parameter is the file path，So i think  `--config` should be deprecated 
or replaced with `--file` here.
I'll modify here later.